app/src/main/java/com/nitrr/ecell/esummit/ecellapp/activities/HomeActivity.java

```diff
@@ -27,14 +21,15 @@
     private RecyclerView recyclerView;
     private HomeRecyclerViewAdapter adapter;
     private List<HomeRVData> homeRVDataList = new ArrayList<>();
+    BottomSheetFragment bottomSheet;
```

Comment:
BottomSheetFragment?? 
Why is this under misc folder not in fragments folder?

Reply:
Done!